spec/data-model/README.md

```diff
@@ -172,46 +188,58 @@ The `Literal` and `VariableRef` correspond to the the _literal_ and _variable_ s
 When they are used as the `body` of an `Expression`,
 they represent _expression_ values with no _annotation_.
 
-`Literal` represents all literal values, both _quoted_ and _unquoted_.
-The presence or absence of quotes is not preserved by the data model.
-The `value` of `Literal` is the "cooked" value (i.e. escape sequences are processed).
+`Literal` represents all literal values, both numeric (_unquoted_) and string (_quoted_).
+The presence or absence of quotes (`'|'` in MessageFormat 2) is not preserved by the data model.
+The `value` of `StringLiteral` is the "cooked" value (i.e. escape sequences are processed).
+The `value` of `NumberLiteral` is the numeric value after parsing (for example the trailing zeros are not preserved).
```

Comment:
```suggestion
The `value` of `NumberLiteral` is the numeric value after parsing,
(for example the trailing zeros are not preserved).
```

Use semantic line breaks.

More substantively: making this the "parsed" numeric value makes it hard to test the roundtrip property (maybe I missed an issue where this was discussed?)